web/src/components/storage/PartitionPage.tsx

```diff
@@ -486,19 +496,20 @@ function SizeOptions({ mountPoint, target }: SizeOptionsProps): React.ReactNode
         </SelectOption>
       )}
       {mountPoint !== NO_VALUE && target !== NEW_PARTITION && (
-        <SelectOption value={NO_VALUE} description={sprintf(_("Keep %s size"), target)}>
+        // TRANSLATORS: %s is a partition name like /dev/vda2
+        <SelectOption value={NO_VALUE} description={sprintf(_("Keep size of %s"), target)}>
           <SizeOptionLabel value={NO_VALUE} mountPoint={mountPoint} target={target} />
         </SelectOption>
       )}
       {mountPoint !== NO_VALUE && target === NEW_PARTITION && (
         <>
-          <SelectOption value="auto" description={_("A proper size is automatically calculated")}>
-            <SizeOptionLabel value="auto" mountPoint={mountPoint} target={target} />
-          </SelectOption>
           <SelectOption
-            value="custom"
-            description={_("Define a custom size between a minimum and a maximum")}
+            value="auto"
+            description={_("Let the installer propose a sensible range of sizes")}
```

Comment:
NP: the auto-calculated size might not be a range. 

And mentioning "the installer" sounds a bit weird to me. Maybe something like: "A size is calculated automatically according to the product requirements".

Reply:
What about "Automatically calculate the size of the partition", without mentioning neither installer nor product.

Reply:
I have no strong opinion here about the use of "the installer". At the end, although we avoid (ab)using such word, it's the installer who will perform the action.

If you both agree, I'd  go for the last proposal from Ancor, at least for releasing the code. We have plenty of time for fine-tuning the wording later.